app/grandchallenge/serving/views.py

```diff
@@ -165,7 +165,9 @@ def serve_structured_challenge_submission_form(
     except ChallengeRequest.DoesNotExist:
         raise Http404("Challenge request not found.")
 
-    if request.user.has_perm("challenges.view_challengerequest"):
+    if request.user.groups.filter(
+        name=settings.CHALLENGES_REVIEWERS_GROUP_NAME
+    ).exists():
```

Comment:
Should use object based permissions here.

Reply:
I'm wondering which permission to use, view or change challenge request? So far we only allowed reviewers to download the submission form, but I don't think that makes any sense. I would say that the creator should be able to download it as well, and hence use the view permission. Would you agree?